src/main/java/io/securecodebox/persistence/defectdojo/model/ScanFile.java

```diff
@@ -6,28 +6,30 @@
 
 import lombok.Data;
 
+// TODO: Why we do not have as many annotations as the other models here?
+// TODO: Why does this class does not implement Model?
```

Comment:
if i remember right this is the class used for the multi part form upload, so its not a json model